rayenbou/ticket-bundle/1.0/config/packages/rayenbou_ticket.yaml

```diff
@@ -0,0 +1,7 @@
+rayen_bou_ticket:
+    authentication:
+        url: "%env(RAYENBOU_TICKET_URL)%"
+        username: "%env(RAYENBOU_TICKET_USERNAME)%"
+        password: "%env(RAYENBOU_TICKET_PASSWORD)%"
+    settings:
```

Comment:
empty ? or does the next should be indented ?

Reply:
I'm currently building a bigger side of this bundle that may need other settings , so i've indented to put more key in future